onnxruntime/core/session/inference_session.cc

```diff
@@ -102,15 +102,16 @@ std::atomic<uint32_t> InferenceSession::global_session_id_{1};
 // Only update this version when there is a file format change which will break the compatibilites
 // Once this model version is updated, the kSupportedOrtModelVersions in IsOrtModelVersionSupported
 // below will also need to be updated.
-static constexpr const char* kOrtModelVersion = "1";
+static constexpr const char* kOrtModelVersion = "2";
 
```

Comment:
We should have a list of what changed between each version in the comments here so the history is obvious.

Reply:
Maybe we want to keep the change list in [https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/flatbuffers/schema/README.md](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/flatbuffers/schema/README.md)?

---
In reply to: [510545943](https://github.com/microsoft/onnxruntime/pull/5540#discussion_r510545943) [](ancestors = 510545943)

Reply:
Either. If we store it in the README.md there should be a comment here that points a developer to the README.md and tells them they must update it when incrementing the 

---
In reply to: [511206054](https://github.com/microsoft/onnxruntime/pull/5540#discussion_r511206054) [](ancestors = 511206054,510545943)